microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/services/MicroProfileValidator.java

```diff
@@ -218,7 +221,7 @@ private static String getErrorIfValueTypeMismatch(ItemMetadata metadata, String
 	}
 
 	private static boolean isBooleanString(String str) {
-		return "true".equals(str) || "false".equals(str);
+		return "true".equals(str) || "false".equals(str) || "Y".equals(str) || "YES".equals(str) || "1".equals(str) || "ON".equals(str);

```

Comment:
Can we have the check be case insensitive? If the value is `oN`, `on`, `TrUE`, `yes` etc., according to https://github.com/eclipse/microprofile-config/blob/master/spec/src/main/asciidoc/converters.asciidoc#built-in-converters we should still consider those to be `true`

Reply:
Thanks! Forgot to look into this.